docs/developer/guides/feature-analysis.md

```diff
@@ -0,0 +1,44 @@
+# ⚙️ Feature Analysis
+
+This document will guide you through the important questions you need to ask yourself when you are starting the analysis phase for the creation of a new feature.
+
+:::info
+Before going further in this document it is important that you ensure the feature should be implemented.
+
+Please check the feature relevance with [Should we implement this ?](./should-we-implement-this).
+
+Refer to that document in case you end up with a "Reconsider" or "Maybe not". This is also a good time to involve other team members and to have a discussion on what you expect from this feature, and what value it will provide to the users.
```

Comment:
```suggestion
Refer to that document in case you end up with a "Reconsider" or "Maybe not". This is also a good time to involve other team members and to have a discussion about what you expect from this feature, and what value it will provide to the users.
```